tests/JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj

```diff
@@ -12,8 +12,8 @@
     <ProjectReference Include="..\JustSaying.TestingFramework\JustSaying.TestingFramework.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="AWSSDK.SimpleNotificationService" />
-    <PackageReference Include="AWSSDK.SQS" />
+    <PackageReference Include="AWSSDK.SimpleNotificationService" VersionOverride="3.7.300.5" />
+    <PackageReference Include="AWSSDK.SQS" VersionOverride="3.7.300.5" />
```

Comment:
TIL - does this work with dependabot?

Reply:
Good question, I don't know, would be good to find out. There is also a feature that I can't find where you define versions in groups, and you can say in the package reference "I want the version from group A", can't remember how that works, so doing this for now.

Reply:
I guess we can merge it and if it messes stuff up we can try something else.